### ISSUE TITLE
API Docs Generation doesn't work

### ISSUE THREAD

**Issue by mhellmeier** [eclipse-tractusx/eclipse-tractusx.github.io#166]
The generation of API documentation with `npm run docusaurus gen-api-docs all` doesn't work and currently results in the following error:
```
OpenAPI docs plugin ID must be specified when more than one plugin instance exists.
```

This is caused by multiple occurrences of the `docusaurus-plugin-remote-content`, e.g.:
https://github.com/eclipse-tractusx/eclipse-tractusx.github.io/blob/main/docusaurus.config.js#L62
https://github.com/eclipse-tractusx/eclipse-tractusx.github.io/blob/main/docusaurus.config.js#L73

They should be merged into one to solve the problem.

**Comment by danielmiehle** [comment]
@maximilianong / @santiagoejm can you please check?

**Comment by Siegfriedk** [comment]
@mhellmeier out of curiosity, i assume this got auto closed because you mentioned it in your PR in a 'wrong' way which indicated to github that this issue and the pr is directly connected right?

**Comment by mhellmeier** [comment]
@Siegfriedk Totally right! 😉

**Comment by Siegfriedk** [comment]
Fixed in #194

**Comment by Siegfriedk** [comment]
@mhellmeier it worked for me. Can you update the issue with the error message / reproduce steps?

**Comment by mhellmeier** [comment]
It can't work since there are still three problems in the config file:
1. `/openApi/bpdm/pool.yaml` and `/openApi/bpdm/pool.yaml` doesn't exist
2. The `Template Kit` folder doesn't exist
3. The traceability open-apis are missing

Would be nice to know which commands you are using to get it working.

Steps to reproduce:
1. Check out the `main` branch
5. Run `npm run docusaurus gen-api-docs all`
6. Error:
```
Loading of api failed for "[...]/eclipse-tractusx.github.io/openApi/bpdm/pool.yaml"
[ERROR] Error: ENOENT: no such file or directory, lstat '[...]/eclipse-tractusx.github.io/openApi/bpdm/pool.yaml'

```

I fixed it in #232. Further, there are still Open API specifications missing. I addressed it in #233.
